src/Repository/AdherentMessageRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\AdherentMessage\AbstractAdherentMessage;
+use AppBundle\Entity\AdherentMessage\AdherentMessageInterface;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+class AdherentMessageRepository extends ServiceEntityRepository
+{
+    use UuidEntityRepositoryTrait;
+
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, AbstractAdherentMessage::class);
+    }
+
+    public function findAllForAuthor(Adherent $adherent): array
+    {
+        return $this->findBy(['author' => $adherent]);
+    }
+
+    public function markAsSynchronized(AdherentMessageInterface $message): void
+    {
+        $message->setSynchronized(true);
+
+        $this->getEntityManager()->flush();
```

Comment:
IMO it's not the responsability of a repository to flush, what do you think to call the `flush()` in `AdherentMessageChangeCommandHandler` ?

Reply:
done